ARMeilleure/CodeGen/X86/HardwareCapabilities.cs

```diff
@@ -24,6 +28,27 @@ static HardwareCapabilities()
                 FeatureInfo7Ebx = (FeatureFlags7Ebx)ebx7;
                 FeatureInfo7Ecx = (FeatureFlags7Ecx)ecx7;
             }
+
+            Xcr0InfoEax = (Xcr0FlagsEax)GetXcr0Eax();
+        }
+
+        private static uint GetXcr0Eax()
+        {
+            byte[] asmGetXcr0 = {
```

Comment:
```suggestion
            ReadOnlySpan<byte> asmGetXcr1 = new byte[] {
```
Compiler optimization to avoid the allocation

Reply:
This still allocates, you'd need to stackalloc it to avoid the allocation. This optimisation only works when the variable is a statically declared member of a static class.

Reply:
I stand corrected. It seems this optimisation has been expanded to support lifting function scoped variables declared `ReadOnlySpan` into static members.